### PR TITLE
4 columns Maintenance status page

### PR DESCRIPTION
**Pull request by arhimede**

### CODE REVIEW

**Comment by arhimede** [comment]
Preview here:
https://preview-lf246gy-2ja7ciew2nbkm.us-2.platformsh.site/packages-maintenance-status/

File: src/App/Handler/MaintenanceOverviewHandler.php [hunk 1]
```diff
@@ -15,6 +15,23 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
...
+        'laminas-mvc-plugin-identity',
+        'laminas-mvc-plugin-prg',
+        'laminas-mvc-middleware',
+        'laminas-mvc-i18n',
+        'laminas-mvc-psr7bridge',
+        'laminas-mvc-console',
+        'laminas-test',
+        'laminas-mvc-form',
+        'laminas-mvc-plugins',
+        'laminas-mvc-view',
```
Comment (froschdesign) [inline src/App/Handler/MaintenanceOverviewHandler.php:32]: No release yet, but okay! 👍🏻

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@arhimede 
Sorry, found a bug: https://github.com/laminas-mvc/laminas-modulemanager

**Review by froschdesign — CHANGES_REQUESTED** [review]
All links for the MVC packages are wrong.

File: src/App/Handler/MaintenanceOverviewHandler.php [hunk 2]
```diff
@@ -15,6 +15,23 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
...
+    public const array MVC_COMPONENTS = [
+        'laminas-mvc',
+        'laminas-modulemanager',
+        'laminas-mvc-plugin-fileprg',
+        'laminas-mvc-plugin-flashmessenger',
+        'laminas-mvc-plugin-identity',
+        'laminas-mvc-plugin-prg',
+        'laminas-mvc-middleware',
+        'laminas-mvc-i18n',
+        'laminas-mvc-psr7bridge',
```
Comment (Xerkus) [inline src/App/Handler/MaintenanceOverviewHandler.php:27]: No such package. psr7bridge is not a mvc component.

File: src/App/Handler/MaintenanceOverviewHandler.php [hunk 3]
```diff
@@ -15,6 +15,23 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
     public const string CUSTOM_PROPERTIES_FILE      = 'maintenance-status.json';
     public const string CUSTOM_PROPERTIES_DIRECTORY = '/public/share';
 
+    public const array MVC_COMPONENTS = [
+        'laminas-mvc',
+        'laminas-modulemanager',
```
Comment (Xerkus) [inline src/App/Handler/MaintenanceOverviewHandler.php:20]: module manager is not a mvc component

Comment (arhimede) [reply src/App/Handler/MaintenanceOverviewHandler.php:20]: @froschdesign and @Xerkus let's produce a list of all MVC -related components , to correct this filter

Comment (froschdesign) [reply src/App/Handler/MaintenanceOverviewHandler.php:20]: But listed under MVC for years: https://docs.laminas.dev/mvc/

Comment (froschdesign) [reply src/App/Handler/MaintenanceOverviewHandler.php:20]: I will update the documentation overviews and move laminas-modulemanager and laminas-psr7bridge to components.

Comment (arhimede) [reply src/App/Handler/MaintenanceOverviewHandler.php:20]: Ok, give me 30 minutes :-)

File: src/App/Handler/MaintenanceOverviewHandler.php [hunk 4]
```diff
@@ -24,12 +24,9 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
         'laminas-mvc-plugin-prg',
         'laminas-mvc-middleware',
         'laminas-mvc-i18n',
-        'laminas-mvc-psr7bridge',
+        'laminas-psr7bridge',
         'laminas-mvc-console',
         'laminas-test',
-        'laminas-mvc-form',
-        'laminas-mvc-plugins',
-        'laminas-mvc-view',
```
Comment (froschdesign) [inline src/App/Handler/MaintenanceOverviewHandler.php]: Why Why were these removed?

File: src/App/Handler/MaintenanceOverviewHandler.php [hunk 5]
```diff
@@ -24,12 +24,9 @@ class MaintenanceOverviewHandler implements RequestHandlerInterface
         'laminas-mvc-plugin-prg',
         'laminas-mvc-middleware',
         'laminas-mvc-i18n',
-        'laminas-mvc-psr7bridge',
+        'laminas-psr7bridge',
```
Comment (froschdesign) [inline src/App/Handler/MaintenanceOverviewHandler.php:27]: Is not a MVC component.

File: src/App/Handler/MaintenanceOverviewHandler.php (hunk 4 above)
Comment (froschdesign) [reply src/App/Handler/MaintenanceOverviewHandler.php]: Everything that starts with "laminas-mvc" is an MVC component.

File: src/App/Handler/MaintenanceOverviewHandler.php (hunk 5 above)
Comment (froschdesign) [reply src/App/Handler/MaintenanceOverviewHandler.php]: The [laminas-developer-tools](https://github.com/laminas/laminas-developer-tools) are missing.

Comment (arhimede) [reply src/App/Handler/MaintenanceOverviewHandler.php]: added

**Comment by froschdesign** [comment]
The name of the subprojects should be used as titles for the columns; Mezzio, Components, MVC, API Tools:

- https://docs.laminas.dev
- https://getlaminas.org

File: src/App/Handler/MaintenanceOverviewHandler.php (hunk 5 above)
Comment (arhimede) [reply src/App/Handler/MaintenanceOverviewHandler.php]: https://preview-lf246gy-2ja7ciew2nbkm.us-2.platformsh.site/packages-maintenance-status/

**Review by froschdesign — APPROVED** [review]
